examples/advanced/main.tf

```diff
@@ -24,58 +24,7 @@ module "appid" {
   existing_kms_instance_guid    = var.existing_kms_instance_guid
   skip_iam_authorization_policy = false
   kms_encryption_enabled        = true
-  #  resource_keys                 = [{ name = "${var.prefix}-sc" }]  tracking issue https://github.ibm.com/GoldenEye/issues/issues/8118
-}
-
-########################################################################################################################
-# Manage authentications
-########################################################################################################################
-
-resource "ibm_appid_idp_cloud_directory" "cd" {
-  tenant_id                           = module.appid.tenant_id
-  is_active                           = true
-  identity_confirm_access_mode        = "OFF"
-  identity_field                      = "userName"
-  reset_password_enabled              = false
-  reset_password_notification_enabled = false
-  signup_enabled                      = false
-  self_service_enabled                = false
-  welcome_enabled                     = true
-}
-
-resource "ibm_appid_mfa" "mf" {
-  tenant_id = module.appid.tenant_id
-  is_active = true
-}
-
-########################################################################################################################
-# Add users to the Cloud Directory
-########################################################################################################################
-
-resource "random_password" "password" {
-  length           = 16
-  special          = true
-  override_special = "!#-"
-}
-
-resource "ibm_appid_cloud_directory_user" "user" {
-  depends_on = [ibm_appid_idp_cloud_directory.cd] # For deletion user must be deleted before the Cloud Directory
-
-  tenant_id = module.appid.tenant_id
-
-  email {
-    value   = "attendee-1@example.com"
-    primary = true
-  }
-
-  active = true
-
-  user_name = "${var.prefix}-${format("%02d", 1)}"
-  password  = random_password.password.result
-
-  display_name = "${var.prefix}-${format("%02d", 1)}"
-
-  create_profile = true
+  resource_keys                 = [{ name = "${var.prefix}-sc" }]
```

Comment:
can you update this so it creates different kinds of keys. Writer / Manager / Reader. but also a key using an existing service ID (you can create one on the example and use it).

Reply:
addresses in https://github.com/terraform-ibm-modules/terraform-ibm-appid/pull/145